Sources/MapboxNavigation/CPRouteChoice.swift

```diff
@@ -0,0 +1,34 @@
+import CarPlay
+import MapboxDirections
+
+@available(iOS 12.0, *)
+extension CPRouteChoice {
+    
+    struct RouteResponseUserInfo {
+        
+        static let key = "com.mapbox.navigation.cpRouteChoice.routeResponse"
+        
+        /**
+         Route response from the Mapbox Directions service.
+         */
+        let response: RouteResponse
+        
+        /**
+         Index of the route.
+         */
+        let routeIndex: Int
+        
+        /**
+         Options, which were used for calculating results from the Mapbox Directions service.
+         */
+        let options: DirectionsOptions
```

Comment:
Not sure why we exposed `options` separately in tuple, as it's already available in `response`. Will probably remove it, so that `RouteResponseUserInfo` only contains `routeResponse` and `routeIndex` properties.